cypress/e2e/toolbar.cy.ts

```diff
@@ -17,7 +17,13 @@ describe('Toolbar', () => {
     const widthIndexToSelect = 1;
 
     assertFramesMatch(frames);
-    selectWidthPreferenceByIndex(widthIndexToSelect);
+
+    const widthSizeToSelectAsNumber = parseInt(
+      frames[widthIndexToSelect].replace('px', ''),
+      10
+    );
+
```

Comment:
Given we convert back to a string in `selectWidthPreference` I feel like this parsing is not necessary.

Reply:
I think it makes sense to have the `width` type as `number` in `selectWidthPreference` to make it clearer that it shouldn't get the 'px' string. Although it is a bit awkward to use here with how this test is set up. Thoughts?

Reply:
I think we can agree that this logic shouldnt be in the test, so then the question becomes where should it be.

The config for `widths` takes an array of numbers, its in the Frames component that we add the `px` suffix.

So maybe we change the above `frames` array to an array of numbers, and we append the `px` in `assertFramesMatch`? That way the api for that util accepts the same structure as the playroom config

Thoughts?

Reply:
Ahh yes good idea. I've moved the formatting logic into the `assertFramesMatch` function.